lib/index.js

```diff
@@ -48,41 +77,133 @@ function compile_template(template_name, type) {
  * the email has been sent.
  */
 function sendemail(template_name, person, callback) {
-  // see: http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/SES.html#sendEmail-property
-  var params = {
-    Destination: { /* required */
-      ToAddresses: [
-        person.email
-      ]
-    },
-    Message: { /* required */
-      Body: { /* required */
-        Html: {
-          Data: compile_template(template_name, 'html')(person), /* required */
-          Charset: 'utf8'
+    // see: http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/SES.html#sendEmail-property
+    var params = {
+        Destination: { /* required */
+            ToAddresses: [
+                person.email
+            ]
         },
-        Text: {
-          Data: compile_template(template_name, 'txt')(person), /* required */
-          Charset: 'utf8'
+        Message: { /* required */
+            Body: { /* required */
+                Html: {
+                    Data: compile_template(template_name, 'html')(person),
+                    /* required */
+                    Charset: 'utf8'
+                },
+                Text: {
+                    Data: compile_template(template_name, 'txt')(person),
+                    /* required */
+                    Charset: 'utf8'
+                }
+            },
+            Subject: { /* required */
+                Data: person.subject,
+                /* required */
+                Charset: 'utf8'
+            }
+        },
+        Source: person.senderEmailAddress || process.env.SENDER_EMAIL_ADDRESS,
+        /* required */
+        ReplyToAddresses: [
+            person.replyToAddress || process.env.SENDER_EMAIL_ADDRESS
+        ]
+    };
+    ses.sendEmail(params, function(err, data) {
+        if (err) { console.log('SES ERROR:', err, err.stack); } // an error occurred
+        return callback(err, data);
+    });
+
+}
+
+
+/**
+ * sendemail method takes a template name and person array object for the To's, CC's & BCC and uses
+ * AWS SES to send the desired email.
+ * @param {String} template_name - the template to use for the email
+ * @param {Object} template_objects - the objects that will be replaced in template dynamically {{token}}
+ * @param {String} subject - the subject of the email
+ * @param {Object} person - the object containing the details of the
+ * person to whom we want to send the email. Requires both name
+ * and email. if you don't *know* the name of the person, leave it
+ * an empty string.
+ * @param {Function} callback - continuation function called after
+ * the email has been sent.
+ */
```

Comment:
I don't understand the distinction between `sendemail` and `sendemails` ...
does one send ***multiple*** emails and the other ***single***?